docs/getting-started.md

```diff
@@ -139,6 +139,24 @@ Now that you're setup, if you can run it on the command line, you can run it in
     4. For `Bazel Command` you can put any bazel command, like `build`, `run`, `test`, etc.
     5. Click `Ok`, then there should be a green arrow in the top right corner by the drop-down menu. Click it and the test will run!
 
+### Running our AI, Simulator or Robot Diagnostics
+
+1. Open your terminal and run `ifconfig`.
+2. Pick the network interface you would like to use:
+    1. If you are running things locally, you can pick any interface that is not `lo`
+    2. If you would like to communicate with robots on the network, make sure to select the interface that is connected to the same network as the robots.
```

Comment:
How do we know which interface is connected to the robots? This is more of a general question for me as I'm not too sure how the different interfaces are used

Reply:
Usually most computers only have 1 or 2 interfaces. You can run `ifconfig` and you will see all of the available interfaces on the left column. The one with `lo` is the loopback interface which is internal to your computer so you never want to use that one. 

The other two (you may only have 1 for WiFi if your computer doesn't have ethernet) you can infer from the interface name. 

Usually wlp3s0, or wlan0 are used for wireless interfaces and enp6s0, eth0 are used for wired interfaces (the numbers can be different). If you have more than 1, and they are not clear from the name, you can just use trial and error at that point, you won't see the robots in the visualizer if the interface doesn't work